Philips.CodeAnalysis.Test/Verifiers/DiagnosticVerifier.cs

```diff
@@ -40,6 +43,25 @@ protected void VerifyDiagnostic(string source, string filenamePrefix, params Dia
 			VerifyDiagnosticsInternal(new[] { source }, filenamePrefix, analyzer, expected);
 		}
 
+		protected void VerifyDiagnostic(string source)
```

Comment:
Alternatively this could be implemented as an extension method on `SingleDiagnosticAnalyzer`

Reply:
I'll play with this idea.